src/main/java/com/aws/greengrass/shadowmanager/ShadowManager.java

```diff
@@ -211,130 +214,225 @@ void handleDeviceThingNameChange(WhatHappened whatHappened, Node changedNode) {
 
     @Override
     protected void install() {
+        install(InstallConfig.builder()
+                .configureMaxDocSizeLimitConfig(true)
+                .configureRateLimitsConfig(true)
+                .configureStrategyConfig(true)
+                .configureSyncDirectionConfig(true)
+                .configureSynchronizeConfig(true)
+                .installDatabase(true)
+                .build());
```

Comment:
Unit tests should just test a unit of code right? I needed to write too many mocks and returns to test a small part of the code. So adding this helped reduce that a bit.

Reply:
So the reason I set it to false, is because most of the times the function is called with one field set to true and the rest as false. So that is clear what part of the `install` function we are trying to test.
Are you suggesting I remove this and have all the mocks in tests?

Reply:
Spoke offline. We will extract out the configuration parsing into a separate class with specific methods like `parseSyncDirection`. This will enable a better coding pattern. I will handle this in the next PR